Formula/m/mbpoll.rb

```diff
@@ -0,0 +1,29 @@
+class Mbpoll < Formula
+  desc "Command-line utility to communicate with ModBus slave (RTU or TCP)"
+  homepage "https://epsilonrt.fr"
+  url "https://github.com/epsilonrt/mbpoll/archive/refs/tags/v1.5.2.tar.gz"
+  sha256 "7d960cd4459b5f7c2412abc51aba93a20b6114fd75d1de412b1e540cfb63bfec"
+  license "GPL-3.0-only"
+  head "https://github.com/epsilonrt/mbpoll.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libmodbus" => ["3.1.4"]
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+    bin.install "build/mbpoll"
+  end
+
+  test do
+    # NOTE: using "1.0-0" and not "1.5.2" due to open PR: https://github.com/epsilonrt/mbpoll/pull/58
+    assert_match "1.0-0", shell_output("#{name} -V")
+
+    invalid_request = "#{name} -1 -o 0.01 -q -m tcp invalid.host 2>&1"
+    assert_empty shell_output(invalid_request, 1)
+    assert_match "Connection failed", pipe_output(invalid_request)
```

Comment:
How can the same command be empty and return "Connection failed"?

Reply:
The first test uses `shell_output` to test the return code (expects 1, and not the default 0)
The second test matched the returned string